package.json

```diff
@@ -4,12 +4,18 @@
   "description": "WordPress Theme framework",
   "main": "index.js",
   "scripts": {
+	"bs": "browser-sync start --config build/browser-sync.config.js",
     "css": "npm-run-all css-compile css-postcss css-minify",
     "css-compile": "sass --style expanded --source-map --embed-sources --no-error-css sass/theme.scss:css/theme.css sass/custom-editor-style.scss:css/custom-editor-style.css",
     "css-minify": "cleancss -O1 --format breakWith=lf --with-rebase --source-map --source-map-inline-sources --output css/ --batch --batch-suffix \".min\" \"css/*.css\" \"!css/*.min.css\" \"!css/*rtl*.css\"",
-    "css-postcss": "postcss --config postcss.config.js --replace \"css/*.css\" \"!css/*.rtl*.css\" \"!css/*.min.css\"",
-    "watch": "npm-run-all --parallel watch-*",
-    "watch-css": "nodemon --watch sass/ --ext scss --exec \"npm-run-all css\""
+    "css-postcss": "postcss --config build/postcss.config.js --replace \"css/*.css\" \"!css/*.rtl*.css\" \"!css/*.min.css\"",
+    "js": "npm-run-all js-compile js-minify",
+    "js-compile": "rollup --config build/rollup.config.js --sourcemap",
+    "js-minify": "terser --compress typeofs=false --mangle --comments \"/^!/\" --source-map \"content=js/theme.js.map,includeSources,url=theme.min.js.map\" --output js/theme.min.js js/theme.js",
+    "watch": "npm-run-all --parallel watch-run-*",
+	"watch-bs": "npm-run-all --parallel bs watch-run-*",
+    "watch-run-css": "nodemon --watch sass/ --ext scss --exec \"npm-run-all css\"",
```

Comment:
nodemon is not a dev dependency in package.json and do we need nodemon at all?
This is per see not a node.js app

Reply:
I'm using nodemon to watch our asset files for changes and re-run all of the compile/minify/etc scripts. Open to alternatives though

Reply:
rollup.js' `watch` command maybe `-w, --watch                 Watch files in bundle and rebuild on changes`

Reply:
Oh that's cool - but I'd need something for the scss?

Reply:
rollup-plugin-scss

Otherwise it works just like webpack where you include your scss in index.js and extract it

Reply:
In that case I'll stick with the default way Bootstrap handles it, using nodemon. 

I'm wondering if it's worth moving the sass folders though, it's always bothered me and people we teach that there's `/sass/` and `/src/sass/`...

Reply:
all scss and js folders should be in /src

Reply:
> In that case I'll stick with the default way Bootstrap handles it, using nodemon.
> 
> I'm wondering if it's worth moving the sass folders though, it's always bothered me and people we teach that there's `/sass/` and `/src/sass/`...

just remember to add nodemon as a dev dep then.

Of course do you, I just think it's a cleaner approach to have the jobs the tasks run defined in a config file for that purpose ie. rollup.config.js